librocksdb-sys/build.rs

```diff
@@ -206,14 +206,14 @@ fn build_rocksdb() {
             "port/win/win_logger.cc",
         ]);
 
-        if cfg!(feature = "jemalloc") {
+        if cfg!(all(feature = "jemalloc", not(windows))) {
             lib_sources.push("port/win/win_jemalloc.cc");
```

Comment:
I don't get it. Why do we need to add a source file related to the windows on `not(windows)` target?

Reply:
yeah good point, this is pointless